notification/src/api/notification/notification.controller.js

```diff
@@ -0,0 +1,133 @@
+const _ = require('lodash')
+const Promise = require('bluebird')
+const httpUtils = require('../../utils/commons')
+const config = require('../../config/config').load()
+const ctp = require('../../utils/ctp')
+const ctpClient = ctp.get(config)
+
+const adyenEvents = require('../../../resources/adyenEvents')
+
+// TODO: add JSON schema validation:
+// https://github.com/commercetools/commercetools-adyen-integration/issues/9
+async function handleNotification (request, response, logger) {
+  const body = await httpUtils.collectRequestData(request, response)
+  try {
+    await processNotifications(JSON.parse(body), logger)
+    response.statusCode = 200
+    return response.end("{ notificationResponse : '[accepted]' }")
+  } catch(e) {
+    logger.error(e, 'Ooops')
+    response.statusCode = 500
+    return response.end()
+  }
+}
+
+// TODO: proper naming
+async function processNotifications (request, logger) {
+  const notifications = request.notificationItems
+  await Promise.map(notifications, notification => processNotification(notification, logger))
```

Comment:
https://github.com/commercetools/commercetools-adyen-integration/commit/f9074861dbb68e4247c088632f4eacb89d73ecbe

> How do you handle this?

 in this case I'll just save an interfaceInteraction since the transaction status is already on a payment.